stories/ui-cohort-date-range/cohort-date-range.stories.ts

```diff
@@ -95,6 +94,31 @@ const COHORTS_ACTIVE: TsDateCohort[] = [
   },
 ];
 
+const COHORTS_WITH_CUSTOM_DATES_ACTIVE: TsDateCohort[] = [
+  {
+    display: 'Last 90 days',
+    range: {
+      start: startOfDay(subDays(DATES.start, 90)),
+      end: DATES.start,
+    },
+  },
+  {
+    display: 'Last full month',
+    range: {
+      start: startOfDay(subMonths(startOfMonth(DATES.end), 1)),
+      end: endOfDay(subDays(startOfMonth(DATES.end), 1)),
+    },
+  },
+  {
+    display: 'Custom Dates',
```

Comment:
looks like should be `TS_CUSTOM_DATES_DESCRIPTION`, no ?

Reply:
no, stories and libs it is different directories

Reply:
i see imports from library 

consumers should e able to import it

Reply:
done